influunt-app/app/views/controladores/wizard/wizard.html

```diff
@@ -3,20 +3,6 @@
     <div class="col-lg-12 col-sm-12 no-padding">
       <div class="ibox float-e-margins">
         <div class="steps-wrapper wizard clearfix">
-          <div class="steps clearfix">
-            <ul class="progress-indicator" wizard-steps>
-              <li ui-sref=".dados_basicos({id: $state.params.id})" ng-class="{ active: $state.current.name === 'app.wizard_controladores.dados_basicos' }"> <span class="bubble"></span>{{ 'controladores.dadosBasicos' | translate }}</li>
-              <li ui-sref=".aneis({id: $state.params.id})" ng-class="{ active: $state.current.name === 'app.wizard_controladores.aneis' }"> <span class="bubble"></span>{{ 'controladores.aneis' | translate }}</li>
-              <li ui-sref=".configuracao_grupo({id: $state.params.id})" ng-class="{ active: $state.current.name === 'app.wizard_controladores.configuracao_grupo' }"> <span class="bubble"></span>{{ 'controladores.configuracao_grupo' | translate }}</li>
-              <li ui-sref=".verdes_conflitantes({id: $state.params.id})" ng-class="{ active: $state.current.name === 'app.wizard_controladores.verdes_conflitantes' }"> <span class="bubble"></span>{{ 'controladores.verdes_conflitantes' | translate }}</li>
-              <li ui-sref=".associacao({id: $state.params.id})" ng-class="{ active: $state.current.name === 'app.wizard_controladores.associacao' }" > <span class="bubble"></span>{{ 'controladores.associacao' | translate }}</li>
-              <li ui-sref=".transicoes_proibidas({id: $state.params.id})" ng-class="{ active: $state.current.name === 'app.wizard_controladores.transicoes_proibidas' }"> <span class="bubble"></span>{{ 'controladores.transicoesProibidas' | translate }}</li>
-              <li ui-sref=".atraso_de_grupo({id: $state.params.id})" ng-class="{ active: $state.current.name === 'app.wizard_controladores.atraso_de_grupo' }"> <span class="bubble"></span>{{ 'controladores.atrasoDeGrupo' | translate }}</li>
-              <li ui-sref=".entre_verdes({id: $state.params.id})" ng-class="{ active: $state.current.name === 'app.wizard_controladores.entre_verdes' }"> <span class="bubble"></span>{{ 'controladores.entreVerdes' | translate }}</li>
-              <li ui-sref=".associacao_detectores({id: $state.params.id})" ng-class="{ active: $state.current.name === 'app.wizard_controladores.associacao_detectores' }"> <span class="bubble"></span>{{ 'controladores.associacaoDetectores.titulo' | translate }}</li>
-              <li ui-sref=".revisao({id: $state.params.id})" ng-class="{ active: $state.current.name === 'app.wizard_controladores.revisao' }"> <span class="bubble"></span>{{ 'controladores.revisao' | translate }}</li>
-            </ul>
-          </div>
```

Comment:
Isso também não esta sendo removido de todas as outras telas? O Wizard só não deve aparecer na tela de revisão quando for visualizar o controlador.

Durante o processo de revisão no wizard esse "wizard" deve aparecer